src/main/resources/db_scripts/postgres-rutherford-create-script.sql

```diff
@@ -132,6 +132,7 @@ CREATE TABLE public.gameboards (
     id character varying NOT NULL,
     title text,
     questions character varying[],
+    contents jsonb[] DEFAULT array[]::jsonb[] NOT NULL,
```

Comment:
I now need a migration script that updates the contents field for all gameboards from its question field. If you want to have a go at this, James, feel free! I'll do it otherwise